src/PhpSpecExtension.php

```diff
@@ -21,6 +21,7 @@ public function load(ContainerBuilder $container, array $config)
         $loader->load('services.xml');
 
         $container->setParameter('phpspec_extension.path', $config['path']);
+        $container->setParameter('phpspec_extension.config', $config['config']);
```

Comment:
I'm getting a warning about this index being missing as the default is null and I'm not setting it in the config file. Can it default to the usual location?